showcase/public/assets/styles/@hashicorp/themed-tokens/with-css-selectors--advanced/tokens.css

```diff
@@ -0,0 +1,621 @@
+/**
+ * Do not edit directly, this file was auto-generated.
+ */
+
+:root,
+.hds-theme-default {
+  --token-color-palette-neutral-0: #ffffff;
+  --token-color-foreground-primary: #ffffff;
+  --token-typography-font-weight-regular: 400;
+  --token-typography-font-weight-medium: 500;
+  --token-typography-font-weight-semibold: 600;
+  --token-typography-font-weight-bold: 700;
+  /* added these to test theming in the showcase */
+  --token-color-foreground-strong: #0c0c0e;
+  --token-color-foreground-primary: #3b3d45;
+  --token-color-surface-strong: #f1f2f3;
+  --token-color-surface-primary: #ffffff;
+  --token-color-surface-faint: #ffffff;
+  --token-typography-font-stack-display:
+    -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif,
+    Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol;
+  --token-typography-font-stack-text:
+    -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif,
+    Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol;
+}
```

Comment:
All these overrides are added to test the different themes application in the `foundations/theming` page. See screenshot below to understand what I mean:

<img width="911" height="1006" alt="Image" src="https://github.com/user-attachments/assets/2b8fc977-e008-49ee-bab6-52008d754851" />